rfcs/text/0000-matrix-service.md

```diff
@@ -0,0 +1,406 @@
+
+#### Metadata
+
+*Related tables:* `analysis`, `donor`, `specimen`, `cell_suspension`, `library_preparation`, `project`, `publication`, `contributor`
+
+The majority of metadata is parsed directly from metadata JSON files available in primary bundles. These fields are identifiable as fields with a valid _Canonical Metadata Names_ mapping in _Table 2.1_. The exceptions are "derived fields" such as `derived_organ_*` that require a degree of intellectual reasoning of the experiment to determine an accurate value. For example, the value of `derived_organ_ontology` may differ from the value of `organ_ontology`, which is read directly from `specimen_from_organism.organ.ontology`, as it is determined by the presence of organoids and/or cell lines in the experiment.
+
+Metadata entities are identified by their `*.provenance.document_id` (DIDs) which are consistently used as primary keys in the schema defined in *Table 2.1*. A similar set of assumptions applied to _Cell IDs_ described in *Table 2.2* apply to these IDs as well:
+
+1. DIDs are unique for distinct metadata entities.
+2. DIDs are preserved across all bundles updates to preserve referential integrity of metadata entities over time.
+3. DIDs are versioned so that metadata can be synchronized across DCP services.
+
+A discussion should be had with the DCP regarding the feasibility of DIDs meeting these assumptions. (1) is potentially at risk as it is possible for the same metadata entity to have different DIDs in separate projects. (2) may be true and requires confirmation from Wranglers/Ingest. Work to satisfy (3) is ongoing and tracked [here](https://github.com/HumanCellAtlas/ingest-central/issues/598).
```

Comment:
Could pop this into the Unresolved Questions section.

Reply:
Great idea!